includes/class-wc-payments-order-service.php

```diff
@@ -518,6 +519,25 @@ public function get_charge_id_for_order( $order ) : string {
 		return $order->get_meta( self::CHARGE_ID_META_KEY, true );
 	}
 
+	/**
+	 * Get any dispute data for specified order.
+	 *
+	 * @param  mixed $order The order Id or order object.
+	 *
+	 * @return array|null The dispute data, or null if no dispute found.
+	 *
+	 * @throws Order_Not_Found_Exception
```

Comment:
```suggestion
	 * @throws Order_Not_Found_Exception
     * @throws WCPay\Core\Exceptions\Server\Request\Extend_Request_Exception
	 * @throws WCPay\Core\Exceptions\Server\Request\Immutable_Parameter_Exception
	 * @throws WCPay\Core\Exceptions\Server\Request\Invalid_Request_Parameter_Exception
```

I see the above exceptions might be thrown by `$request->send()`.